### PR TITLE
bugfix: return the path of video file with data

### PR DESCRIPTION
**Pull request by SABlyu**
copy file path to local variable before calling re-initialization method, which override private class field with new file path (with zero data in it)

### Bugs Fixed ###
there was a bug, which prevented to get a genuine video file path on the upper level of camera view. 
here is the bug report: https://github.com/xamarin/XamarinCommunityToolkit/issues/355
- Fixes #
save the path of the file with data to a local variable to return it when the data is there
### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/xamarin/XamarinCommunityToolkit?pullRequest=356) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/xamarin/XamarinCommunityToolkit?pullRequest=356) before we can accept your contribution.<br/><br/>:x: SABlyu [sign now](https://cla.dotnetfoundation.org/xamarin/XamarinCommunityToolkit?pullRequest=356)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/xamarin/XamarinCommunityToolkit?pullRequest=356) it.</sub>

**Comment by jfversluis** [comment]
Looks like it's failing to build: 

> XamarinCommunityToolkit\Views\CameraView\Android\CameraFragment.android.cs(445,20): Error CS0103: The name 'fileName' does not exist in the current context

**Comment by jfversluis** [comment]
@SABlyu could you please let us know if you're able to bring this home? Thanks!

File: XamarinCommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs [hunk 1]
```diff
@@ -437,10 +437,13 @@ public async void StopRecord()
 		{
 			if (IsBusy || !IsRecordingVideo || session == null || mediaRecorder == null)
 				return;
+			var capturedFile = string.Empty;
 
 			try
 			{
 				DisposeMediaRecorder();
+				capturedFile = fileName;
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs:445]: What is "fileName" here?

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]
`fileName` not exists in that context, something is missing?

**Comment by AndreiMisiukevich** [comment]
Closing this PR due to not having a response from the author for a long time.
Also, this PR has a compile error.

**Pull request closed without merging**